fjord/__init__.py

```diff
@@ -49,3 +58,11 @@ def setup_vendor_path():
 
 
 setup_vendor_path()
+
+monkeypatch()
+
+
+# Setup celery
+# This will make sure the app is always imported when
+# Django starts so that shared_task will use this app.
+from .celery import app as celery_app
```

Comment:
I'm pretty sure we only need this for `celery -A ...` launching. Let's nix this for now.